epochtalk/epochtalk
# Moderation

Added ability to add moderators to a board via the boards management view in the admin panel.

Testing Procedure:
* Visit the boards management view
* Click the person icon to the right of the board name
* Test adding moderators
* Test removing moderators
* Test adding/removing at the same time
* Once a mod is added test that they can only moderate the board they are listed as moderator of (Assuming the user isn't a super admin/admin/global mod)
* Ensure that the Admin->Management->Moderators view is no longer there
